dtfs-central-api/api-tests/v1/tfm/facilities/tfm-facility-put.api-test.js

```diff
@@ -54,11 +52,13 @@ describe('/v1/tfm/facilities', () => {
         .put({
           dealType: CONSTANTS.DEALS.DEAL_TYPE.BSS_EWCS,
           dealId: '61e54e2e532cf2027303e001',
-          auditDetails: generatePortalAuditDetails(MOCK_PORTAL_USER._id),
+          auditDetails: portalAuditDetails,
         })
         .to('/v1/tfm/deals/submit');
 
-      const { status } = await api.put({ facility: newFacility, user: MOCK_PORTAL_USER }).to('/v1/tfm/facilities/61e54e2e532cf2027303e001');
+      const { status } = await api
+        .put({ facility: newFacility, user: MOCK_PORTAL_USER, auditDetails: portalAuditDetails })
```

Comment:
I think this should be `tfmAuditDetails` - its the PUT facilties/:id endpoint, not PUT facilities/id/snapshot.